Frontend/android-native/mobility-common/src/main/java/in/juspay/mobility/common/MobilityCommonBridge.java

```diff
@@ -606,7 +624,9 @@ protected void updateLastKnownLocation(String callback, boolean animate, final S
                             if (animate && googleMap != null) {
                                 LatLng latLng = new LatLng(lat, lng);
                                 if (userPositionMarker == null) {
-                                    upsertMarker(CURRENT_LOCATION, String.valueOf(lat), String.valueOf(lng), 160, 0.5f, 0.9f); //TODO this function will be removed
+                                    String marker = "";
+                                    if (!EDIT_LOCATION) marker = CURRENT_LOCATION;
```

Comment:
instead of adding this check we can send a flag from purescript which would decide if we want the marker to be drawn or not.

Reply:
should line no.629 also be a part of if condition?